packages/docusaurus-theme-classic/codeTranslations/fa.json

```
@@ -59,4 +59,4 @@
   "theme.tags.tagsListLabel": "برچسب ها:",
   "theme.tags.tagsPageLink": "مشاهده تمام برچسب ها",
   "theme.tags.tagsPageTitle": "برچسب ها"
-}
+}
```

Let's add new line at end of file as was before

```suggestion
}

```

---

https://github.com/facebook/docusaurus/pull/5640/commits/c7cce5f6ca031ade49369ba6f7a27a4c7e7567fa

You're so helpful 😊